synapse/lib/heap.py

```diff
@@ -56,17 +87,20 @@ def __init__(self, fd, **opts):
         size = fd.tell()
```

Comment:
While you're in here, line 83 is the last uncovered line. Can we add/modify a test so that this also gets hit?

Reply:
I'll take a look - i don't see why not.